src/app/index.spec.tsx

```diff
@@ -45,6 +45,24 @@ describe('create app', () => {
     expect(app.history.location.state).toEqual('asdf');
   });
 
+  describe('inside the browser', () => {
+    jest.unmock('../helpers/Sentry');
```

Comment:
instead of restoring this mock it would probably be better to create a new mock with the spy you need so you don't accidentally trigger any of that underlying error capture functionality